src/utils.cpp

```diff
@@ -226,6 +300,32 @@ void TextChat::resetTag()
 {
     tag = -1;
 }
+
+bool TextChat::setNodeUserAccess(handle h, handle uh, bool revoke)
+{
+    if (revoke)
+    {
+        attachments_map::iterator uhit = attachedNodes.find(h);
+        if (uhit != attachedNodes.end())
+        {
+            uhit->second.erase(uh);
+            if (uhit->second.empty())
+            {
+                attachedNodes.erase(h);
+                changed.attachments = true;
+                return true;
```

Comment:
Shouldn't the function return true if the `erase` succeeded even if other users have still access to the node?